docs/doc/21-load-data/00-local.md

```diff
@@ -72,7 +72,7 @@ CREATE TABLE books
 <TabItem value="csv" label="CSV">
 
 ```shell title='Request'
-curl -XPUT 'http://root:@127.0.0.1:8081/v1/streaming_load' -H 'insert_sql: insert into book_db.books format CSV' -H 'skip_header: 0' -H 'field_delimiter: ,' -H 'record_delimiter: \n' -F 'upload=@"./books.csv"'
+curl -XPUT 'http://root:@127.0.0.1:8081/v1/streaming_load' -H 'insert_sql: insert into book_db.books format CSV' -H 'format_skip_header: 0' -H 'format_field_delimiter: ,' -H 'format_record_delimiter: \n' -F 'upload=@"./books.csv"'
```

Comment:
So our SQL is not changed?

Reply:
no,  copy do not need prefix `FILE_FORMAT = (type = 'CSV' field_delimiter = '\t')`